### DISCUSSION TITLE
Maybe a bug?

### DISCUSSION THREAD

**Discussion by ghost** [Maplespe/DWMBlurGlass#353] [Q&A] [answered]
Hey, so this is a question for anyone using Windows 10 Enterprise LTSC 1809 (Build: 17763.5936) I have installed today DWM BlurGlass and it doesn't want to work and trows me out this error everytime i try to install or boot up my computer. I post it here due to i don't want to waste any dev time if it's something i have misunderstood due to my crappy English. Greetings, Kosmo.
![DWM](https://github.com/Maplespe/DWMBlurGlass/assets/69810562/b9684269-e5fa-46fa-8e97-219bea571f46)

**Comment by ALTaleX531** [comment, marked as answer]
Nope, we've pointed out in the README that the lowest supported version is Windows 10 2004, the system you're using is NOT SUPPORTED.
